dbms/src/Storages/DeltaMerge/BitmapFilter/BitmapFilter.h

```diff
@@ -28,7 +28,7 @@ class BitmapFilter
     void set(BlockInputStreamPtr & stream);
     void set(const ColumnPtr & col, const FilterPtr & f);
     void set(const UInt32 * data, UInt32 size, const FilterPtr & f);
-    void set(UInt32 start, UInt32 limit);
+    void set(UInt32 start, UInt32 limit, bool value = true);
```

Comment:
Maybe use different names for these three `set(...)`? I'm afraid we may run into calling unexpected overload because of the params change or default value in the signature one day.

Reply:
Now they are safe because there are no similarities in arguments. We need to rename if add a similar signature or let's do this in a separate PR. This PR is kind of big now.

Reply:
1. `void set(const UInt32 * data, UInt32 size, const FilterPtr & f);` -> `set(std::span<const UInt32> data, const FilterPtr & f);`
2. `void set(const ColumnPtr & col, const FilterPtr & f);` and `void set(std::span<const UInt32> data, const FilterPtr & f);` as private.